submissions/Eugene-CG/DOM/style.css

```diff
@@ -0,0 +1,97 @@
+:root {
+  --full-size: 100%;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.delegate-events-container {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+}
+.wrapper {
+  min-height: 100vh;
+  max-width: var(--full-size);
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  grid-template-columns: 1fr 2fr 2fr;
+  grid-template-areas:
+    "header header header"
+    "main main main"
+    "footer footer footer";
+  background-image: url(./img/gitlogoforme.jpeg);
```

Comment:
[Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/background-repeat) about `background-repeat` property, mabey it makes your design much better.